book/templates.rst

```diff
@@ -295,6 +295,38 @@ property, set the attribute ``mandatory`` to ``true``:
         </properties>
     </template>
 
+Language Independent Properties
+-------------------------------
+
+Some content like article numbers or other metadata might be the same for every
+language. In that case, you can mark properties as not multilingual using
+``multilingual="false"``:
+
+.. code-block:: xml
+
+    <?xml version="1.0" ?>
+    <template xmlns="http://schemas.sulu.io/template/template"
+              xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+              xsi:schemaLocation="http://schemas.sulu.io/template/template http://schemas.sulu.io/template/template-1.0.xsd">
+        <!-- ... -->
+
+        <properties>
+            <!-- ... -->
+
+            <property name="article_number" type="text_line" multilingual="false">
+                <!-- ... -->
+            </property>
+
+            <!-- ... -->
+        </properties>
+    </template>
+
+.. note::
+
+    If you change an existing attribute from a multilingual to a non multilingual version,
+    it is necessary to migrate the values from one language to the new attribute,
```

Comment:
Also use `property` instead of `attribute`. And I would mention `the values in PHPCR` or something like that, since that is not completely clear.